apps/channels/datamodels.py

```diff
@@ -93,6 +93,31 @@ def parse(message_data: dict) -> "TwilioMessage":
         )
 
 
+class SureAdhereMessage(BaseModel):
+    """
+    A wrapper class for user messages coming from the sureadhere
+    """
+
+    chat_id: int
+    message_text: str | None
+    content_type: MESSAGE_TYPES | None = Field(default=MESSAGE_TYPES.TEXT)
+    content_type_unparsed: str | None = Field(default=None)
+
+    @field_validator("content_type", mode="before")
+    @classmethod
+    def determine_content_type(cls, value):
```

Comment:
Since `content_type` is always `TEXT` I think this can be removed along with `content_type_unparsed`.